docs/runtime_api.md

```diff
@@ -353,9 +353,13 @@ any.typeUrl; // type.googleapis.com/docs.User
 any.is(User); // true
 any.is(Timestamp); // false
 
+// Unpack an Any using a type registry
+const typeRegistry = createRegistry(User);
+any.unpack(typeRegistry); // Message of type User
+
 // Unpack an Any by providing a blank instance:
-message = new User();
-any.unpackTo(message); // true
```

Comment:
These 3 lines actually belong to a "story" we were telling: 1. check what the Any contains 2. unpack by providing blank instance.

Can you change the order and move "Unpack using type registry" below "unpack by blank instance"? Might also change comment to read "Alternative, unpack an Any using a type registry:"